src/nft/components/explore/Banner.tsx

```diff
@@ -91,12 +90,24 @@ const HeaderContainer = styled.div`
   }
 
   @media only screen and (max-width: ${({ theme }) => `${theme.breakpoint.sm}px`}) {
-    font-size: 20px;
-    line-height: 28px;
-    justify-content: center;
-    align-items: center;
-    padding-top: 0px;
-    padding-bottom: 0px;
+    line-height: 43px;
+    text-align: center;
+    padding-bottom: 16px;
+
+    br {
+      display: none;
+    }
+  }
+
+  /* Custom breakpoint to split into two lines on smaller screens */
+  @media only screen and (max-width: 550px) {
```

Comment:
can we use a themed breakpoint here

Reply:
Unfortunately, we don't have a breakpoint for this. It's a one-off specifically calculated at a approximate width where the text would start breaking into two lines

Reply:
For the record, I added `xs` breakpoint as that is actually the one that exists in Figma and is being referenced across different layouts!